lib/core/repository/cms/attribute_builder.rb

```diff
@@ -12,23 +12,21 @@ def self.build(response)
       end
 
       def attributes
-        attributes = response.body
+        attrs = response.body
```

Comment:
@Tr4pSt3R I think you're confusing your notes from our pairing session. My comment on not overloading variable names was based on a piece of code where you reused a variable to contain two different values (that I don't like). This was fine.